resources/views/tasks/edit.blade.php

```diff
@@ -0,0 +1,150 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="col-sm-offset-2 col-sm-8">
+            <!-- Current Tasks -->
+            @if ($task)
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        @lang('task.task')
+                    </div>
+
+                    <div class="panel-body">
+                        @include('common.errors')
+
+                        <table class="table table-striped task-table" id="table-task">
+                            <thead>
+                                <th>@lang('task.name')</th>
+                                <th>@lang('task.description')</th>
+                            </thead>
+                            <tbody>
+                                <td>
+                                    <div>{{ $task->name }}
+                                        <div>
+                                </td>
+                                <td>
+                                    <div>{{ $task->description }}</div>
+                                </td>
+                                <td>
+                                    <button type="submit" class="btn btn-danger" id="button-edit">
+                                        <i class="fa fa-pencil"></i>
+                                    </button>
+                                </td>
+                            </tbody>
+                        </table>
+                        <form action="{{ route('tasks.update', ['task' => $task->id]) }}" method="POST" id="form-edit"
+                            class="form-horizontal" style="display:none">
```

Comment:
```suggestion
                            class="form-horizontal">
```
Không viết style trong view